style.css

```diff
@@ -39,15 +42,99 @@
     margin-bottom: 12px;
   }
 
-  .humburger {
-    color: #fff;
-    padding-right: 36px;
+  .logo.active {
+    color: rgba(0, 0, 0, 0.05);
+    cursor: default;
   }
 
   .navi-menu {
+    display: flex;
+    list-style-type: none;
+    position: fixed;
+    left: -100%;
+    height: 100%;
+    top: 3rem;
+    flex-direction: column;
+    background-color: #3c3a39;
+    width: 100%;
+    border-radius: 10px;
+    text-align: center;
+    transition: 0.3s;
+    box-shadow: 0 10px 27px rgba(0, 0, 0, 0.05);
+  }
+
+  .navi-menu.active {
+    left: 0;
+  }
+
+  .navi-link {
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+    text-decoration: none;
+    text-align: left;
+    padding-left: 16px;
+    margin-top: 30px;
+    border-bottom: 1px solid #6f6c6b;
+    margin-left: 24px;
+  }
+
+  .material-icons {
+    text-decoration: none;
+    padding-bottom: 26px;
+    padding-left: 15px;
+    color: #fff;
+    height: 4px;
+    width: 12px;
+    font-size: 25px;
+    margin-left: 30px;
```

Comment:
- [ ] Great job so far, Just one small thing is left which is important while following the Figma design.
As you can see below video you have to follow the same colour for menu item options (i.e. #FFF5E1 ) and for the close (X) button (i.e. #67798E).

https://user-images.githubusercontent.com/70269507/195112183-f5867580-db37-4d67-a47e-22a0b1539610.mov